packages/@glimmer/interfaces/lib/compile/index.d.ts

```diff
@@ -1,4 +1,4 @@
-export * from './encoder.js';
-export * from './instruction-encoder.js';
-export * from './operands.js';
-export * from './wire-format/api.js';
+export type * from './encoder.js';
```

Comment:
Even though this is a `.d.ts` file, it seemed fine to follow the linting rules and `export type`.
The whole `@glimmer/interfaces` story could probably use a clear plan and overhaul, but I think it's
fine to follow the normal TypeScript lints here unless there's a good reason not to.